types.d.ts

```diff
@@ -1362,7 +1362,7 @@ declare namespace mage {
          */
         interface IUserCommand {
             acl?: string[];
-            execute(state: IState, ...args: any[]): any;
+            execute<T>(state: IState, ...args: any[]): Promise<T>;
```

Comment:
I actually want to use the type argument to explicitly type (or infer) the return value of the execute method.